src/Features/Lsif/Generator/Program.cs

```diff
@@ -43,7 +44,19 @@ private static async Task GenerateAsync(FileInfo? solution, FileInfo? project, F
         {
             // If we have an output file, we'll write to that, else we'll use Console.Out
             using var outputFile = output != null ? new StreamWriter(output) : null;
-            var outputWriter = outputFile ?? Console.Out;
+            TextWriter? outputWriter = null;
+
+            if (outputFile is null)
+            {
+                Console.Out.Encoding = UTF8Encoding.UTF8;
+                Console.OutputEncoding = UTF8Encoding.UTF8;
```

Comment:
What's the difference between these two?

Reply:
lol, errors were too slow, turns out we only have OutputEncoding available (others are unsettable). Updating